### ISSUE TITLE
A Quick Introduction to Backpropagation Softmax Layer Little Error

### ISSUE THREAD

**Issue by kli-casia** [arunmallya/arunmallya.github.io#2]
![qq 20161129204045](https://cloud.githubusercontent.com/assets/13194870/20710169/217fe07a-b674-11e6-9c26-1d791a1364cd.jpg)

**Comment by arunmallya** [comment]
Has been reported in #1 , will fix by weekend (fingers crossed). Thanks for reminding me.

**Comment by arunmallya** [comment]
Btw, this has been fixed. Thanks!
